src/NUnitEngine/nunit.engine.tests/Services/ServiceManagerTests.cs

```diff
@@ -9,63 +9,63 @@ namespace NUnit.Engine.Services.Tests
         private ServiceManager _serviceManager;
 
-        private IService _projectService;
+        private IService1 _service1;
+        private IService2 _service2;
 
         [SetUp]
         public void SetUp()
         {
             _serviceManager = new ServiceManager();
 
-            _settingsService = new FakeSettingsService();
-            _serviceManager.AddService(_settingsService);
-
-            _projectService = new Fakes.FakeProjectService();
-            _serviceManager.AddService(_projectService);
+            _service1 = new FakeService1();
+            _serviceManager.AddService(_service1);
+            _service2 = new FakeService2();
+            _serviceManager.AddService(_service2);
         }
 
         [Test]
         public void InitializeServices()
         {
             _serviceManager.StartServices();
 
-            IService service = _serviceManager.GetService(typeof(ISettings));
+            IService service = _serviceManager.GetService(typeof(IService1));
             Assert.That(service.Status, Is.EqualTo(ServiceStatus.Started));
-            service = _serviceManager.GetService(typeof(IProjectService));
+            Assert.That(service, Is.SameAs(_service1));
+            service = _serviceManager.GetService(typeof(IService2));
             Assert.That(service.Status, Is.EqualTo(ServiceStatus.Started));
+            Assert.That(service, Is.SameAs(_service2));
         }
 
         [Test]
         public void InitializationFailure()
         {
-            ((FakeSettingsService)_settingsService).FailToStart = true;
-            Assert.That(() => _serviceManager.StartServices(), 
-                Throws.InstanceOf<InvalidOperationException>().And.Message.Contains("FakeSettingsService"));
+            ((FakeService)_service1).FailToStart = true;
+            Assert.That(() => _serviceManager.StartServices(),
+                Throws.InstanceOf<InvalidOperationException>().And.Message.Contains("FakeService1"));
         }
 
         [Test]
         public void TerminationFailure()
         {
-            ((FakeSettingsService)_settingsService).FailedToStop = true;
-            _settingsService.StartService();
+            ((FakeService)_service1).FailToStop = true;
+            _service1.StartService();
 
             Assert.DoesNotThrow(() => _serviceManager.StopServices());
         }
 
         [Test]
         public void AccessServiceByClass()
         {
-            IService service = _serviceManager.GetService(typeof(FakeSettingsService));
-            Assert.That(service, Is.SameAs(_settingsService));
+            IService service = _serviceManager.GetService(typeof(FakeService2));
+            Assert.That(service, Is.SameAs(_service2));
         }
 
-        [Test]
-        public void AccessServiceByInterface()
-        {
-            IService service = _serviceManager.GetService(typeof(ISettings));
-            Assert.That(service, Is.SameAs(_settingsService));
-        }
```

Comment:
Did you mean to remove this test, rather than refactor in the same way as above?

Reply:
I realized it duplicated a check made in the InitializeServices test.